docs/installation-guide/quickstart.rst

```diff
@@ -81,15 +84,17 @@ Prepare the bootstap node
    apiServer:
```

Comment:
for minion variable, shouldn't we say: hostname-of-the-bootstrap-node ?

Reply:
I'll try to detail that part - either use the hostname, or set `/etc/salt/minion_id`